conference/migrations/0006_auto_20180927_1103.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
let's use descriptive names for migration files. please. ;)

Reply:
Good point - updated!